decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/_promoted_process_group.html.erb

```diff
@@ -1,31 +1 @@
-<% cta_settings = participatory_process_group_cta_settings(promoted_process_group) %>
-<div class="card card--full card--process">
-  <div class="row collapse card--process__row">
-    <div class="columns mediumlarge-8 large-6 card--process__column">
-      <div class="card__content">
-        <%= link_to participatory_process_group_path(promoted_process_group), class: "card__link" do %>
-          <h2 class="card__title"><%= decidim_html_escape(translated_attribute(promoted_process_group.title)).html_safe %></h2>
-        <% end %>
-        <%= decidim_sanitize_editor html_truncate(translated_attribute(promoted_process_group.description), length: 630, separator: "...") %>
-        <%= link_to participatory_process_group_path(promoted_process_group), class: "button small hollow card__button" do %>
-          <span class="show-for-sr"><%= decidim_html_escape(translated_attribute(promoted_process_group.title)) %></span>
-          <%= t("participatory_processes.promoted_process_group.more_info", scope: "layouts.decidim") %>
-        <% end %>
-      </div>
-    </div>
-    <div class="columns mediumlarge-8 large-6 card--process__column">
-      <div class="card--full__image" style="background-image:url('<%= cta_settings&.image_url || promoted_process_group.attached_uploader(:hero_image).path %>')">
-        <% if cta_settings.present? %>
-          <div class="card__content row collapse">
-            <div class="large-6 large-offset-6 columns">
-              <%= link_to cta_settings.path, class: "button expanded button--sc" do %>
-                <span class="show-for-sr"><%= cta_settings.text %></span>
-                <span aria-hidden="true"><%= cta_settings.text %></span>
-              <% end %>
-            </div>
-          </div>
-        <% end %>
-      </div>
-    </div>
-  </div>
-</div>
+<%= card_for promoted_process_group, highlight: true, size: :g %>
```

Comment:
This is only being used in ```decidim/decidim-participatory_processes/app/cells/decidim/participatory_processes/content_blocks/highlighted_processes/single_process.erb:8```
Maybe just put the celll in that partial?

Reply:
Yes I've replaced this usage with a call to highlighted card-g and simplified to use the card regardless the class of the resource. This content block is not fully redesigned yet, this will be completed in a future PR, but this is a first approach